src/Components/Search/NewSearch/NewSearchBar.tsx

```diff
@@ -0,0 +1,23 @@
+import { Text } from "@artsy/palette"
+import { ClassI18n } from "System/i18n/ClassI18n"
+import styled from "styled-components"
+
+export const NewSeacrhBar = () => {
+  return (
+    <ClassI18n>
+      {({ t }) => (
+        <Form
+          action="/search"
+          method="GET"
+          onSubmit={() => console.log("Pressed ")}
+        >
+          <Text> I am the new Search bar</Text>
```

Comment:
From "Acceptance Criteria" section for [FX-4651](https://artsyproduct.atlassian.net/browse/FX-4651)

> And user typed 2 or more characters in the search input
> User sees a new search dropdown component (which is empty for now)

if it's not difficult, can you please add this too as part of the current PR? or did you want to add this as part of the next PR?

Reply:
Ya sure. I am updating the PR

Reply:
thanks!